packages/now-next/src/utils.ts

```diff
@@ -174,7 +174,8 @@ async function getRoutes(
   entryDirectory: string,
   pathsInside: string[],
   files: Files,
-  url: string
+  url: string,
+  isDev?: boolean
```

Comment:
```suggestion
  url: string
```
Same as https://github.com/vercel/vercel/pull/4510/files#r441644891